### PR TITLE
Feature/Tests to Generated Projects

### PR DESCRIPTION
**Pull request by 20pa5a1210**
added tests to generated projects. just used standard go testing library because using testify or any other 3rd party libraries
need to install that package we don't know user want a testing library or not. i mean most people usually don't write tests tho. so sticking with standard testing good instead of using a 3rd party one

### CODE REVIEW

File: cmd/program/program.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ type Templater interface {
 	Main() []byte
 	Server() []byte
 	Routes() []byte
+    TestHandler() []byte
```
Comment (Melkeydev) [inline cmd/program/program.go:40]: nit: format?

File: cmd/program/program.go [hunk 2]
```diff
@@ -49,6 +50,7 @@ var (
 
 	cmdApiPath         = "cmd/api"
 	internalServerPath = "internal/server"
+    testHandlerPath    = "tests"
```
Comment (Melkeydev) [inline cmd/program/program.go:53]: nit: format

**Review by Melkeydev — APPROVED** [review]
This looks dope - i have a few nits that I will leave up to you to address.
And OMG tests??!?!?!?!!?!?

**Comment by 20pa5a1210** [comment]
Fixed Format nits

**Pull request merged**